components/ui-shell/settings.ts

```diff
@@ -0,0 +1,149 @@
+export interface NavItem {
+  label: string;
+  url?: string;
+  children?: NavItem[];
+  segment?: SegmentData;
+}
+
+export interface TopLevelNavItem extends NavItem {
+  isMegaMenu?: boolean;
+}
+
+export interface SegmentData {
+  cta: string;
+  location: string;
+}
+
+export enum Variant {
+  DEFAULT = '',
+  WITH_ACCOUNT = 'with-account',
+}
+
+export const NAV_ITEMS: TopLevelNavItem[] = [
+  {
+    label: 'Overview',
+    url: 'https://qiskit.org/overview/',
+  },
+  {
+    label: 'Learn',
+    url: 'https://qiskit.org/learn/',
+  },
+  {
+    label: 'Community',
+    children: [
+      {
+        label: 'Events',
+        url: 'https://qiskit.org/events/',
+      },
+      {
+        label: 'Advocates',
+        url: 'https://qiskit.org/advocates/',
+      },
+    ],
+  },
+  {
+    label: 'Documentation',
+    isMegaMenu: true,
+    url: 'https://qiskit.org/documentation/',
+    children: [
+      {
+        label: 'Get started',
+        children: [
+          {
+            label: 'Overview',
+            url: 'https://qiskit.org/documentation/getting_started.html',
+          },
+        ],
+      },
+      {
+        label: 'Core',
+        children: [
+          {
+            label: 'Qiskit Terra',
+            url: 'https://qiskit.org/documentation/apidoc/terra.html',
```

Comment:
Ideally we should link to the module homepage with overview like ([qiskit-nature](https://qiskit.org/documentation/nature/)) but I know for Terra, Aer and IBMQ provider that are still under the meta package and therefore do not have their individual homepages this is not possible.

But Terra is effectively Qiskit. Perhaps this link should be linking to the Qiskit Documentation homepage: https://qiskit.org/documentation/

With Aer 0.11 release (coming soon), aer documentation will have [it's own page](https://github.com/Qiskit/qiskit/pull/1595). Depending on when this PR is merged, we should change the link to https://qiskit.org/documentation/aer